pod/bridges/GenericChangelogBridge.php

```
@@ -0,0 +1,413 @@
+    {
+        // Look for repeating h2 or h3 headings that resemble release/changelog titles
+        foreach (['h2', 'h3'] as $tag) {
+            $headings = $html->find($tag);
+            if (empty($headings)) {
+                continue;
+            }
+
+            $matchingHeadings = [];
+            foreach ($headings as $h) {
+                $text = trim($h->plaintext);
+                if (preg_match('/(v?\d+\.\d+|\b(20\d\d[-/.][01]\d[-/.][0-3]\d)\b|release|changelog|update)/i', $text)) {
```

PHPStan (2.2.8)

[error] Regex pattern is invalid: Unknown modifier '.' in pattern: /(v?\d+\.\d+|\b(20\d\d[-/.][01]\d[-/.][0-3]\d)\b|release|changelog|update)/i


(regexp.pattern)